src/core/openthread.cpp

```diff
@@ -991,6 +994,25 @@ void HandleActiveScanResult(void *aContext, Mac::Frame *aFrame)
     return;
 }
 
+ThreadError otEnergyScan(uint32_t aScanChannels, uint16_t aScanDuration, otHandleEnergyScanResult aCallback,
```

Comment:
Not sure how do we plan to set other TLVs that is needed by MGMT_ED_SCAN.qry like Count TLV, Commissioner Session TLV here?
I am now working on an OT command that is used to send MGMT commands (PR #402 ). Not sure do we need to use one OT command to transmit all the MGMT commands?
This OT command would have 3 params, the Path to indicate the command type, TLVs and total size.
The cons are that are not very user friendly that user need to fill TLVs by themselves.
Or do we plan to one MGMT command has one OT command?
I think that the pros are that may be more user friendly than one OT command.